packages/migrations/src/batched-migrations/batched-migration-job.ts

```diff
@@ -1,8 +1,10 @@
+import { fileURLToPath } from 'url';
+
 import { z } from 'zod';
 
 import { loadSqlEquiv, queryRows } from '@prairielearn/postgres';
 
-const sql = loadSqlEquiv(import.meta.filename);
+const sql = loadSqlEquiv(fileURLToPath(import.meta.url));
```

Comment:
Just pointing out that `loadSqlEquiv` accepts a URL as parameter. This is in use all over the apps/prairielearn codebase.